src/main/java/uk/gov/hmcts/reform/bulkscanning/exceptions/ServiceConfigNotFoundException.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscanning.exceptions;
+
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+import uk.gov.hmcts.reform.logging.exception.UnknownErrorCodeException;
+
+/**
+ * SonarQube reports as error. Max allowed - 5 parents
+ */
```

Comment:
This is not javadoc comment for this class, it should be a regular comment for the annotation (if it is actually really needed).

(I know this is done in other repos as well)